nautobot/tenancy/views.py

```diff
@@ -18,68 +20,72 @@
 #
 
 
-class TenantGroupListView(generic.ObjectListView):
-    queryset = TenantGroup.objects.all()
-    filterset = filters.TenantGroupFilterSet
-    filterset_form = forms.TenantGroupFilterForm
-    table = tables.TenantGroupTable
-
-
-class TenantGroupView(generic.ObjectView):
+class TenantGroupUIViewSet(
+    mixins.ObjectDetailViewMixin,
+    mixins.ObjectListViewMixin,
+    mixins.ObjectEditViewMixin,
+    mixins.ObjectDestroyViewMixin,
+    mixins.ObjectBulkDestroyViewMixin,
+    mixins.ObjectBulkCreateViewMixin,
+    mixins.ObjectChangeLogViewMixin,
+    mixins.ObjectNotesViewMixin,
+):
```

Comment:
Only need `BulkUpdateViewMixin` to replace these lines with `NautobotUiViewSet`

Reply:
https://github.com/nautobot/nautobot/pull/6865/commits/92388203b4235a21498651cd1b9b7b13fc0a066b